pkg/networkplugin-syncer/handlers/ovn/routers_test.go

```diff
@@ -0,0 +1,31 @@
+package ovn
+
+import (
+	goovn "github.com/ebay/go-ovn"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var portA = "port-a"
+var portB = "port-b"
+
+var _ = Describe("Router routes functions", func() {
+
+	When("Filtering routes", func() {
```

Comment:
I think the `When` is redundant and can be removed.

Reply:
Done